device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/IotHubStatusCode.java

```diff
@@ -123,4 +123,36 @@ public static IotHubStatusCode getIotHubStatusCode(int httpsStatus)
 
         return iotHubStatus;
     }
+
+    /**
+     * Returns true if this event callback signals that the asynchronous action was unsuccessful, but could be retried.
+     * Returns false if it was successful, or it was unsuccessful but should not be retried.
+     * @param statusCode The status code.
+     * @return true if this event callback signals that the asynchronous action was unsuccessful, but could be retried, and false otherwise
```

Comment:
```suggestion
     * @return true if this event callback signals that the asynchronous action was unsuccessful, but could be retried, and false otherwise.
```